presto-orc/src/main/java/com/facebook/presto/orc/OrcReader.java

```diff
@@ -149,15 +150,17 @@ public OrcReader(OrcDataSource orcDataSource, MetadataReader delegate, DataSize
             completeFooterSlice = Slices.wrappedBuffer(buffer, buffer.length - completeFooterSize, completeFooterSize);
         }
 
+        Supplier<LocalMemoryContext> untrackedMemory = new AggregatedMemoryContext().localContextSupplier();
```

Comment:
The intention behind the variable name is to reveal the fact the memory seems to be accounted but the accounting goes to dev null (not reported anywhere). Any `new AggregateMemoryContext()` that is not read by anyone is untracked.